.gitignore

```diff
@@ -22,8 +22,5 @@ test-dist
 .local
 yarn.lock
 
-
-# nixago: ignore-linked-files
-/lefthook.yml
-/.editorconfig
-/treefmt.toml
\ No newline at end of file
+.pre-commit-config.yaml
+jsdelivr-npm-importmap.js
```

Comment:
NOTE: I added `jsdelivr-npm-importmap.js` to gitignore because it is generated by `npm run build` and it causes issues with the pre-commit formatters.

Reply:
Fixed